packages/devtools_server/lib/src/server.dart

```diff
@@ -586,7 +582,17 @@ ArgParser _createArgsParser(bool verbose) {
       hide: !verbose,
     );
 
-  // Hidden args.
+  // Deprecated and hidden args.
+  // TODO: Remove this - prefer that clients use the rest arg.
+  parser
+    ..addOption(
+      argVmUri,
```

Comment:
Do we know whether this can be removed yet? `--vm-uri`

Reply:
@jacob314 do you know if this is used anywhere?

Reply:
I just verified. It is not used in vscode, flutter-intellij or flutter.